docs/semgrep-app/integrations.md

```diff
@@ -0,0 +1,19 @@
+---
+slug: integrations
+append_help_link: true
+description: "Semgrep App contains 3rd party integrations to allow users to add data from Semgrep to other tools that are part of their workflows."
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Integrations
+
+## JIRA
+
+Our JIRA integration allows you to create tickets directly from our Findings page with relevant info about a particular finding.
+
+![Setting up the JIRA integration](../img/jira-integrations-page.png)
+![Creating a JIRA ticket from the Findings page](../img/jira-findings-page.png)
+![Output of JIRA integration](../img/jira-template.png)
```

Comment:
```suggestion
![Setting up the Jira integration](../img/jira-integrations-page.png)
![Creating a Jira ticket from the Findings page](../img/jira-findings-page.png)
![Output of Jira integration](../img/jira-template.png)
```

Also, I suggest leaving a bit more space between the images so they are easier to distinguish. I’m not sure if just adding line breaks will do it however, since the Markdown parser may just swallow those.